testdata/scripts/health/multi-chain-loopp.txtar

```diff
@@ -0,0 +1,414 @@
+env CL_SOLANA_CMD=chainlink-solana
+env CL_STARKNET_CMD=chainlink-starknet
+env CL_MEDIAN_CMD=chainlink-feeds
+env CL_MERCURY_CMD=chainlink-mercury
+
+# start node
+exec sh -c 'eval "echo \"$(cat config.toml.tmpl)\" > config.toml"'
+exec chainlink node -c config.toml start -p password -a creds &
+
+# initialize client
+env NODEURL=http://localhost:$PORT
+exec curl --retry 10 --retry-max-time 60 --retry-connrefused $NODEURL
+exec chainlink --remote-node-url $NODEURL admin login -file creds --bypass-version-check
+
+exec chainlink --remote-node-url $NODEURL health
+cmp stdout out.txt
+
+exec chainlink --remote-node-url $NODEURL health -json
+cp stdout compact.json
+exec jq . compact.json
+cmp stdout out.json
+
+exec chainlink --remote-node-url $NODEURL health -failing
+cmp stdout out-unhealthy.txt
+
+exec chainlink --remote-node-url $NODEURL health -f -json
+cp stdout compact.json
+exec jq . compact.json
+cmp stdout out-unhealthy.json
+
+-- testdb.txt --
+CL_DATABASE_URL
+-- testport.txt --
+PORT
+
+-- password --
+T.tLHkcmwePT/p,]sYuntjwHKAsrhm#4eRs4LuKHwvHejWYAC2JP4M8HimwgmbaZ
+-- creds --
+notreal@fakeemail.ch
+fj293fbBnlQ!f9vNs
+
+-- config.toml.tmpl --
+[Webserver]
+HTTPPort = $PORT
+
+[[Cosmos]]
+ChainID = 'Foo'
+
+[[Cosmos.Nodes]]
+Name = 'primary'
+TendermintURL = 'http://tender.mint'
+
+[[EVM]]
+ChainID = '1'
+
+[[EVM.Nodes]]
+Name = 'fake'
+WSURL = 'wss://foo.bar/ws'
+HTTPURL = 'https://foo.bar'
+
+[[Solana]]
+ChainID = 'Bar'
+
+[[Solana.Nodes]]
+Name = 'primary'
+URL = 'http://solana.web'
+
+[[Starknet]]
+ChainID = 'Baz'
+
+[[Starknet.Nodes]]
+Name = 'primary'
+URL = 'http://stark.node'
+
+-- out.txt --
+ok Cosmos.Foo.Chain
+ok Cosmos.Foo.Relayer
+ok Cosmos.Foo.Txm
+ok EVM.1
+ok EVM.1.BalanceMonitor
+ok EVM.1.HeadBroadcaster
+ok EVM.1.HeadTracker
+!  EVM.1.HeadTracker.HeadListener
+	Listener is not connected
+ok EVM.1.LogBroadcaster
+ok EVM.1.Relayer
+ok EVM.1.Txm
+ok EVM.1.Txm.BlockHistoryEstimator
+ok EVM.1.Txm.Broadcaster
+ok EVM.1.Txm.Confirmer
+ok EVM.1.Txm.Finalizer
+ok EVM.1.Txm.WrappedEvmEstimator
+ok HeadReporter
+ok JobSpawner
+ok Mailbox.Monitor
+ok Mercury.WSRPCPool
+ok Mercury.WSRPCPool.CacheSet
+ok PipelineORM
+ok PipelineRunner
+ok PipelineRunner.BridgeCache
+ok RetirementReportCache
+!  Solana.Bar.RelayerService
+	plugin unavailable
+!  StarkNet.Baz.RelayerService
+	plugin unavailable
+ok TelemetryManager
+ok WorkflowDBStore
+
+-- out-unhealthy.txt --
+!  EVM.1.HeadTracker.HeadListener
+	Listener is not connected
+!  Solana.Bar.RelayerService
+	plugin unavailable
+!  StarkNet.Baz.RelayerService
+	plugin unavailable
```

Comment:
TODO:
- [ ] are these available in CI?

Reply:
I have managed to get these reporting, but only with a hacky artificial delay inside of the `PluginService.HealthReport()` method (waiting for the plugins to boot up). However, moving this delay to the testscript `.txtar` file does not cause the same effect. :confused: